pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -256,22 +264,15 @@ public RebalanceResult rebalance(TableConfig tableConfig, Configuration rebalanc
           tierToInstancePartitionsMap, null);
     }
 
-    if (currentAssignment.equals(targetAssignment)) {
+    boolean segmentAssignmentUnchanged = currentAssignment.equals(targetAssignment);
+    LOGGER.info("For rebalanceId: {},  segmentAssignmentUnchanged: {}, "
```

Comment:
Let's include the table name, and follow the sequence of `instancePartitionsUnchanged`, `tierInstancePartitionsUnchanged`, `segmentAssignmentUnchanged`

Reply:
(minor) The above comment is not addressed. Also there are double spaces

Reply:
I did add the table name (look below). But I'll fix the double spaces